### ISSUE TITLE
Support for asyncio

### ISSUE THREAD

**Issue by possemeeg** [mdowds/python-mock-firestore#76]
Hi, do you guys want a PR with support for asyncio?

**Comment by gocarer** [comment]
@possemeeg, asyncio support would be useful. Do you have an implementation for it?

**Comment by possemeeg** [comment]
Yes. In fact I created a fork from your repo because I needed it: https://github.com/pckmsolutions/firestore-mock. I'm happy to create a PR off your repo if you like.

**Comment by jkp** [comment]
I'm using the fork - thank you! Would be fantastic to get this back into the trunk.
